src/content/configuration/dev-server.md

```diff
@@ -659,6 +659,29 @@ T> [`watchOptions`](#devserver-watchoptions-) will have no effect when used with
 
 T> If you use the CLI, make sure __inline mode__ is disabled.
 
+## `devServer.liveReload`
+
+`boolean: true`
+
+When `devServer.liveReload` is disabled, the dev-server will never reload/refresh the page.
```

Comment:
`never` appears to be a bit awkward.

Reply:
@wizardofhogwarts i updated the description with more detailed one, can you please check it ? thanks.